scripts/android-config.gradle

```diff
@@ -17,4 +17,7 @@ android {
                 '-opt-in=io.customer.base.internal.InternalCustomerIOApi',
         ]
     }
+    testOptions {
+        unitTests.includeAndroidResources = true
```

Comment:
We have this already added in `scripts/android-module-testing.gradle`. Do we still need this again?

Reply:
Missed it there.